README.md

```diff
@@ -80,15 +81,28 @@ Only the TFTs listed below are currently supported. Trying to install the firmwa
 
 **WARNING:** BTT does not officially provide MKS TFT hardware support. MKS TFT is maintained by open source contributors and BTT does not bear any risk of MKS TFT hardware using this firmware.
 
+## Software Repository
+
+Firmware source: https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware
+
+Main branch: **Master**
+
+</br>In the **Master** branch you will find:
+
+- The most recent source code updates
+- The most recent resources such as precompiled firmwares, themes, configuration and language files
+
+**IMPORTANT NOTE:** The **Master** branch is currently the **ONLY** branch in the project to be used. The other currently existing branches **develop** and **release-xx.27** are outdated and **MUST NOT** be used.
+
 ## Marlin Dependencies
 
 Firmware source: https://github.com/MarlinFirmware/Marlin/releases
 
-Minimum Marlin firmware version: 2.0.8.1
+Minimum Marlin firmware version: **2.0.8.1**
 
-Distribution date: 2021-05-15
+Distribution date: **2021-05-15**
 
-In order the TFT firmware is able to provide all of its functionalities/features, the following options must be enabled in Marlin firmware.
+</br>In order the TFT firmware is able to provide all of its functionalities/features, the following options must be enabled in Marlin firmware.
```

Comment:
"In order for the TFT's firmware to be able to provide..."

Reply:
Ok thanks. Applied the change also to config.ini file